src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -715,19 +715,23 @@ public void TestEuroCurrencyFromString(object input, string expected, string lan
         }
 
         [Test]
-        public void TestNullInputCurrency()
+        [TestCase(null)]
+        [TestCase("")]
+        public void TestNullOrEmptyInputCurrency(string input)
         {
             // Set the thread culture and test for backward compatibility
             using (CultureHelper.SetCulture("en-US"))
             {
                 Helper.AssertTemplateResult(
                     expected: string.Empty,
-                    template: "{{ null | currency: 'de-DE' }}");
+                    template: "{{ input | currency: 'de-DE' }}");

```

Comment:
Is there a missing line here defining the hash table with variables? Because this is testing for input variable in scope of Liquid - and not passed from this method.